scalikejdbc-streams/src/main/scala2.12/scalikejdbc/streams/ExecutionContextPreparable.scala

```diff
@@ -0,0 +1,9 @@
+package scalikejdbc.streams
+
+import scala.concurrent.ExecutionContext
+
+trait ExecutionContextPreparable {
```

Comment:
Describe why this wrapper trait is needed with a link to the scaladoc which explains why the method has been marked as deprecated. http://scala-lang.org/api/2.12.1/scala/concurrent/ExecutionContext.html